frontend/src/metabase/actions/hooks/use-action-form/use-action-form.ts

```diff
@@ -40,12 +45,18 @@ function useActionForm({ action, initialValues = {} }: Opts) {
   );
 
   const cleanedInitialValues = useMemo(() => {
-    const values = validationSchema.cast(initialValues);
+    const fieldKeys = action.parameters.map(({ id }) => id);
```

Comment:
Please use `.cast(..., , { stripUnknown: true })` instead to make the schema the single source of truth.